app/views/sessions/index.html.erb

```diff
@@ -1 +1,28 @@
-<h1> Hello! </h1>
\ No newline at end of file
+
+<div class="container">
+    <div class="col-lg-4 col-sm-6 text-center">
+    <div class="well">
+        <h4>What is on your mind?</h4>
+    <div class="input-group">
+        <input type="text" id="userComment" class="form-control input-sm chat-input" placeholder="Write your message here..." />
+      <span class="input-group-btn" onclick="addComment()">     
+            <a href="#" class="btn btn-primary btn-sm"><span class="glyphicon glyphicon-comment"></span> Add Comment</a>
+        </span>
+    </div>
+    <hr data-brackets-id="12673">
+    <ul data-brackets-id="12674" id="sortable" class="list-unstyled ui-sortable">
+        <strong class="pull-left primary-font">James</strong>
+        <small class="pull-right text-muted">
+           <span class="glyphicon glyphicon-time"></span>7 mins ago</small>
+        </br>
+        <li class="ui-state-default">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </li>
+        </br>
+         <strong class="pull-left primary-font">Taylor</strong>
+        <small class="pull-right text-muted">
+           <span class="glyphicon glyphicon-time"></span>14 mins ago</small>
```

Comment:
I'm noticing some inconsistent indentation (3 or 4 spaces, random extra indentation on lines 7 and 20, etc), and trailing whitespace on line 8. Either a 2 or 4 space indent in HTML is acceptable (always 2 spaces in ruby code). Correct and consistent indentation helps a lot with readability/understandability and even increases the perceived code quality to those who read it. Configuring your editor to take care of indentation for you can help a lot.